create-security-certificates.md

```diff
@@ -2,3 +2,68 @@
 title: Create Security Certificates
 toc: false
 ---
+
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` command with the appropriate subcommand and flags. 
```

Comment:
Some advanced users may manage their own CA, and we should say that this is OK: the `cockroach cert` command is not required. We should eventually provide guidelines on how to do this, but don't worry about it for now.